trezorctl

```diff
@@ -1084,6 +1085,44 @@ def ripple_sign_tx(connect, address, file):
     click.echo("Serialized tx including the signature:")
     click.echo(binascii.hexlify(result.serialized_tx))
 
+
+#
+# Tezos functions
+#
+@cli.command(help='Get Tezos address for specified path.')
+@click.option('-n', '--address', required=True, help="BIP-32 path, e.g. m/44'/1729'/0'/0'/0'")
+@click.option('-c', '--curve', default=0, type=click.IntRange(0, 2), help='Curve to use (0 = ed25519, 1 = secp256k1, 2 = p256)')
```

Comment:
please use a `ChoiceType` for curve, see Bitcoin's `get_address` and `CHOICE_INPUT_SCRIPT_TYPE`